src/components/TaskPreviewPanel.js

```diff
@@ -1,68 +1,57 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { injectIntl } from 'react-intl';
+import { withTheme, withStyles } from '@material-ui/core/styles';
 import { Paper, Typography } from '@material-ui/core';
-import { makeStyles } from '@material-ui/styles';
-import TaskPreview from './TaskPreviewTable';
+import {
+  Contributions,
+  withHistory,
+  withModulesManager,
```

Comment:
Since _TaskPreviewPanel_ is a functional component, we should stick to the convention and use _useHistory/useModulesManager_ instead of _withHistory/withModulesManager._
```
import { useHistory, useModulesManager } from '@openimis/fe-core';
  const history = useHistory();
  const modulesManager = useModulesManager();
```

Reply:
done